scripts/cc-cli/src/test/integration-tests/showAddress.test.ts

```diff
@@ -0,0 +1,45 @@
+import execa from "execa";
+import { fundAddressesFromSudo } from "./helpers";
+import { parseAmountInternal } from "../../utils/parsing";
```

Comment:
I see lots of warnings for unused imports. For some reason `eslint` is exiting with 0 here.

Reply:
@pLabarta can you cherry-pick 

https://github.com/gluwa/creditcoin/pull/1259/commits/b17660d8875fcd59d1b5d46f3c2982f47ea57a49

and 

https://github.com/gluwa/creditcoin/pull/1259/commits/9bd8029a9b6c6cb0929cba8b526e7dabca37bab1.


Then it will leave only the warnings here:
```
/home/runner/work/creditcoin/creditcoin/scripts/cc-cli/src/utils/account.ts
Warning:   77:23  warning  Forbidden non-null assertion  @typescript-eslint/no-non-null-assertion
Warning:   78:30  warning  Forbidden non-null assertion  @typescript-eslint/no-non-null-assertion
```